docs/site/Update-generator.md

```diff
@@ -0,0 +1,68 @@
+---
+lang: en
+title: 'Update generator'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Update-generator.html
+---
+
+{% include content/generator-create-app.html lang=page.lang %}
+
+### Synopsis
+
+The `lb4 update` command runs against a LoopBack 4 project and checks
+dependencies against the installed `@loopback/cli`. By default, it checks for
+exact match. Use `--semver` option to check compatibility using semver
+semantics.
+
+```sh
+lb4 update [options]
+```
+
+The key benefit of `lb4 update` over `npm update` is that `lb4 update` honors
+the collection of compatible LoopBack packages (and their dependencies) that are
+released with the current lb4 command from `@loopback/cli`.
+
+If the command is run without a LoopBack 4 project, it prints the version of the
+current CLI and versions of LoopBack packages that are released with the
+`@loopback/cli`. This is the same as `lb4 -v`.
+
+### Options
+
+`--semver` : _(Optional)_ Use semver semantics to check version compatibility
+for project dependencies of LoopBack modules
+
+### Output
+
+The following is a sample output from `lb4 update` against a LoopBack 4 project
+generated by older version of CLI:
+
+```
+The project was originally generated by @loopback/cli@<1.0.1>.
```

Comment:
I'd like you to clarify this point:

What happens after we run `npm update` and then a new version of `@loopback/cli` is published - are we going to print the CLI version that was used to generate the project or the last CLI version that was executed to update dependencies?

For example:

- Project created using `@loopback/cli@<1.0.1>`.
- Later updated by running `@loopback/cli` version `1.2.0`.
- Now I have `@loopback/cli` version `1.3.0` installed and run `lb4 update`.

What will the tool print?

BTW what do the angle brackets around the version mean?